pandas/tests/test_internals.py

```diff
@@ -566,6 +566,13 @@ def test_astype(self):
             else:
                 self.assertEqual(tmgr.get('d').dtype.type, t)
 
+    def test_illegal_arg_for_errors_in_astype(self):
+        """ ValueError exception raised when illegal value used for errors """
```

Comment:
Can you turn this into a normal comment (with `#`)? Otherwise the test name does not show up in the nosetests output. 
You can also add the issue number